src/rmp/src/Restructure.cpp

```diff
@@ -264,17 +264,15 @@ void Restructure::getEndPoints(sta::PinSet& ends,
 
   std::size_t path_found = path_ends->size();
   logger_->report("Number of paths for restructure are {}", path_found);
-  int end_count = 0;
   for (auto& path_end : *path_ends) {
     if (opt_mode_ >= Mode::DELAY_1) {
       sta::PathExpanded expanded(path_end->path(), open_sta_);
       logger_->report("Found path of depth {}", expanded.size() / 2);
       if (expanded.size() / 2 > max_depth) {
         ends.insert(path_end->vertex(sta_state)->pin());
-        end_count++;
       }
-      if (end_count > 5)  // limit blob size for timing
-        break;
+      // Use only one end point to limit blob size for timing
+      break;
```

Comment:
This seems unrelated to typos.  Is this intentional?

Reply:
Yes, there had been feedback to remove 5 as heuristic. With some experimentation done so far, look like keeping it to 1 should be ok. So removing check wrt hard coded value of 5.